exchange/c2cx.com/utils.go

```diff
@@ -105,7 +107,11 @@ func readResponse(r io.ReadCloser) (*response, error) {
 	if err != nil {
 		return nil, err
 	}
-	defer r.Close()
+	defer func() {
+		if err := r.Close(); err != nil {
+			panic(err)
```

Comment:
Probably best to log this case and move on.

Close errors do matter when dealing with files you're writing to though